package.json

```diff
@@ -124,7 +124,82 @@
               }
             }
           }
-        }
+        },
+        "configurationSnippets": [
+          {
+            "label": "Unity Debug: Unity Editor",
+            "body": {
+              "name": "Unity Editor",
+              "type": "unity",
+              "path": "^\"\\${workspaceFolder}/Library/EditorInstance.json\"",
```

Comment:
What does this part do exactly?
I can see that you are quoting the path, is that necessary?
^ <- what does this do in the statement.

Reply:
copied this from vscode mock debugger extension:
https://github.com/microsoft/vscode-mock-debug/blob/a867d18ddbe558e3f0dc3d63f51639d5d8db76d8/package.json#L145

I'm not really familiar with the escape syntax but when I plainly used  '${workspaceFolder}' in my env, it was replaced by empty string when the snippet was added

Reply:
Alright :) Then I have no further comments. Looks good.